homeassistant/components/wink.py

```diff
@@ -66,14 +82,103 @@
 ]
 
 
+def config_from_file(filename, config=None):
```

Comment:
Please don't copy this crap 😉 we should remove this "pattern" from HASS. Make 1 method for reading, 1 for writing.